workshop/content/docs/publishing/ogcapi-features.md

```diff
@@ -92,23 +92,45 @@ This exercise is optional, if time permits, or you can perform this after worksh
 Here we explore publishing OGC API - Features data using pygeoapi with an Elasticsearch backend provider. 
 For this we need the [pygeoapi-examples GitHub repository](https://github.com/geopython/pygeoapi-examples):
 
-<div class="termy">
-```bash
-git clone https://github.com/geopython/pygeoapi-examples.git
-```
-</div>
+=== "Linux/Mac"
+
+    <div class="termy">
+    ```bash
+    git clone https://github.com/geopython/pygeoapi-examples.git
+    ```
+    </div>
+
+=== "Windows (PowerShell)"
+
+    <div class="termy">
+    ```bash
+    git clone https://github.com/geopython/pygeoapi-examples.git
+    ```
+    </div>
 
 Alternatively, you can [download the repo as a .zip file](https://github.com/geopython/pygeoapi-examples/archive/refs/heads/main.zip) and unzip.
 
 Change into the `docker/elastic` folder and run the `docker-compose.yml` file:
 
-<div class="termy">
-```bash
-cd docker/elastic
+=== "Linux/Mac"
 
-docker compose up
-```
-</div>
+    <div class="termy">
+    ```bash
+    cd docker/elastic
+
```

Comment:
In the other terminals, we don't have a line between `cd` and `docker compose`.  See: https://github.com/geopython/diving-into-pygeoapi/pull/185/files#diff-498dbf04e1849ba7fa83e21c99d9d76b1361b5f410c2b1b887eaa1038180d1dcR39